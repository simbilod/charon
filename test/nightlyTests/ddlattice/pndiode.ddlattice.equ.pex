Input FEM file                   = pndiode.ddlattice.equ.exo
LB file                          = pndiode.ddlattice.equ.nem
File Extension for Spread Files  = .exo
Number of Processors             = 4 
------------------------------------------------------------
                Parallel I/O section
------------------------------------------------------------
Parallel Disk Info= number=1, offset=1, zeros, nosubdirectory
Parallel file location = root=/home/xngao/Program/charon/test/ddlattice/, subdir=.
