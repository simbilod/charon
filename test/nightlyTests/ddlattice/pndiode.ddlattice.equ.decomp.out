===========================================================
| Sandia Engineering Analysis Access procedure for: decomp 
===========================================================
nem_slice version 4.04
Performing an elemental load balance with the following parameters...
Machine Description
	architecture: mesh
	dimension(s): 4
	total number of processors: 4
Load Balance Parameters
	type: inertial
	refinement: none
Weighting Parameters
	no weighting
Miscellaneous Options
	SPHERES are being treated as concentrated mass - connectivity exists
	WARNING: side id error checks turned off
ExodusII mesh information
	title: Sierra Output Default Title
	geometry dimension: 2
	number of nodes: 1111	number of elements: 1000
	number of element blocks: 1
	number of node sets: 1	number of side sets: 2
Time to read mesh parameters: 0.000000s
Time to read mesh: 0.000000s
Time to find surrounding elements: 0.000000s
Time to generate graph: 0.000000s
===================Call Chaco===========================
Parameter file `User_Params' not found; using default parameters.


           Input and Parameter Values

# vertices = 1000, # edges = 0
Global method: Inertial
Local method: None
Partitioning target: 1-dimensional mesh of size 4
Partitioning mode: Bisection
Random seed: 1
Active Parameters:
  CHECK_INPUT = True
  OUTPUT_METRICS = 2
  MAKE_VWGTS = False
  REFINE_MAP = False
  REFINE_PARTITION = 0
  INTERNAL_VERTICES = False
  DEBUG_PARAMS = 2



Starting to partition ...

========================================================
Time in Chaco: 0.000000s
Time to generate load balance: 0.000000s
Time for elemental categorization: 0.000000s
Nodal categorization: 0.000000s
Find procs for border nodes: 0.000000s
Order elem cmaps: 0.000000s
Make cmaps consistent: 0.000000s
Time to generate load-balance maps: 0.000000s
Outputting load balance to file pndiode.ddlattice.equ.nem
QA Record:
	nem_slice
	4.04
	24 Oct 2014
	15:28:16
Time to write Nemesis file: 0.020000s
The entire load balance took 0.020000s
